api/server.js

```diff
@@ -514,6 +514,10 @@ async function main(){
         res.status(200).json(await getProjectedGrades(apiAuthClient, req.query['email']));
     })
 
+    app.get('/api/admin/progressquerystring', async (req, res) => {
```

Comment:
Make sure you do error checking.  If you pull main, you'll see what I mean on the previous endpoint you made.